install_config/aggregate_logging.adoc

```diff
@@ -382,6 +383,25 @@ $ oc scale rc/logging-fluentd-1 --replicas=3
 
 You will need to scale Fluentd if nodes are added or subtracted.
 
+[NOTE]
+If the logging project was created using a node selector, then scaling Fluentd will fail. The scheduler will try to place all Fluentd replicas onto the same node. To fix this, edit the project:
```

Comment:
If you use the `====` method for this Note, you don't have to use `+` to link the lines together.